Add INCR type command against wrong argument test cases.

hwware opened the pull request:
We have test cases for incr related commands with no key exist and spaces in key and wrong type of key. However, we dont have test cases covered for INCRBY INCRBYFLOAT DECRBY INCR DECR HINCRBY HINCRBYFLOAT ZINCRBY with valid key and invalid value as argument, and float value to incrby and decrby. So added test cases for the scenarios in incr.tcl.

Thank you!

enjoy-binbin commented on tests/unit/type/incr.tcl:
```diff
@@ -183,6 +183,16 @@ start_server {tags {"incr"}} {
 
+    test {INCRBY INCRBYFLOAT DECRBY against unhappy path} {
+        r del mykeyincr
+        r set mykeyincr 10
```
we can drop this line if we only test the unhappy path
```suggestion
```

hwware replied:
To make sure the key is exist and valid I have added, understand your point and removed now.

enjoy-binbin reviewed (commented):
Have you checked other similar commands like incr, like hincrby?

hwware commented:
> Have you checked other similar commands like incr, like hincrby?

Thank you for notifying @enjoy-binbin , I have added similar cases for remaining INCR type case and updated top comment as well. could you please take look when you have sometime?

zuiderkwast commented on tests/unit/type/hash.tcl:
```diff
@@ -482,6 +482,13 @@ start_server {tags {"hash"}} {
     } {2}
 
+    test {HINCRBY HINCRBYFLOAT against non-integer increment value} {
+        r del incrhash
+        r hset incrhash field 5
+        assert_error "*value is not an integer*" {r hincrby myhash field v}
+        assert_error "*value is not a*" {r hincrbyfloat myhash field v}
```
Typo: `incrhash` is populated but `myshash` is tested.

hwware replied:
Thanks @zuiderkwast , I have updated latest diff with corrected key name.
